src/client/todos/todo.styl

```diff
@@ -0,0 +1,26 @@
+.todo-item
+  font-size .875em
+  margin-bottom .25em
+  max-width 42em
+
+  &__label, &__button
```

Comment:
It's actually a BEM-related stuff, http://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/. Not saying it's great, I am open to any suggestions.